### ISSUE TITLE
Parsing with unknown fields

### ISSUE THREAD

**Issue by kohlivarun5** [patricoferris/ppx_deriving_yaml#39]
I would like to use the ppx to partially define the type, so as to only fetch the needed fields. 

For example, using a type:

```ocaml
type t = {
  type_ : string option; [@key "type"]
}
```

And a yaml:

```yaml
name: vaudit
type: "task"
```

It fails with an error `namevaudit`

Would it be possible to support parsing while skipping any unknown elements in the yaml ?

**Comment by patricoferris** [comment]
Seems useful, fixed in #40

**Comment by patricoferris** [comment]
A release is on its way https://github.com/ocaml/opam-repository/pull/22265
